ROADMAP.md

```diff
@@ -6,6 +6,14 @@ This document provides a "roadmap" to help developers navigate the organization
 
 The `codewit/client` directory contains the react app that drives the front-end.
 
+#### Tailwind Styling Configurations
+
+- All styling configurations for Tailwind are located under `codewit/client/src/tailwind.config.js`
+
+#### Website Icon/Logos
+
+- The site icon, or favicon, which appears in the browser tab, is located under the `codewit/client/public/` directory alongside the logo and hexicon. The icon is currently initialized in `codewit/client/index.html` along with the website title.
```

Comment:
These descriptions are fine, although let's cut out the last sentence. Pretty soon we'll develop a header bar and the favicon will probably "live" there.